### DISCUSSION TITLE
Making '<' input type '\'

### DISCUSSION THREAD

**Discussion by SuperCowProducts** [aptend/typing-transformer-obsidian#75] [Q&A] [answered]
I'm using Typing Transformer to remap keys,  so I want to have the button that would normally output '<' instead output '\\'

I tried a series of different patterns, but they weren't quite what I was after:

- this is almost what I'm looking for, but it puts a space after the '\\' which I don't want
`'<|' -> '\ |'`

- problem is removing the space here will result in a parsing error (the '\\' is parsed as an escape character for '|'):
`'<|' -> '\|'`

- I also tried escaping the space in between (which is necessary to avoid '|' being escaped by '\\') like it's done in bash but now it's just outputting  '\\\ ' without escaping anything! 
`'<|' -> '\\ |'`
    - you can see what I mean about the reference to bash in a cmd like this one, where I navigate in a directory named "A directory": `cd A\ directory/`

Any ideas?

**Comment by aptend** [comment, marked as answer]
Thank you for the thorough explanation and inspiring exploration! The rule `'<|' -> '\\|'` was intended to work correctly, but a bug is preventing it from working. I have filed this in issue #84 and it will be fixed in the next release.

**Comment by aptend** [comment]
New Release! https://github.com/aptend/typing-transformer-obsidian/releases/tag/0.4.2
